### PR TITLE
Sync charge summary panel contents with details below

### PR DESCRIPTION
**Pull request by KentShikama**
This PR removes cases_sorted from RecordSummarizer as a refactoring. The label sorting should just be done in the backend eventually.

### CODE REVIEW

File: src/frontend/src/components/RecordSearch/Record/RecordSummary/ChargesList.tsx [hunk 1]
```diff
@@ -43,12 +39,42 @@ export default class ChargesList extends React.Component<Props> {
...
         return (
-            <li key={"chargeItem" + index} className="bt b--light-gray pt1 mb2"><a href={"#" + chargeName[0]} className="link hover-blue">{chargeName[1]}</a></li>
+            <li key={"chargeItem" + index} className="bt b--light-gray pt1 mb2"><a href={"#" + id} className="link hover-blue">{chargeName}</a></li>
         )
       }));
     return listItems;
   }
+
+  // TODO: This is kind of a hack
+  extractLabel([label, _]: [string, any[]]): string {
```
Comment (wittejm) [inline src/frontend/src/components/RecordSearch/Record/RecordSummary/ChargesList.tsx:52]: And it is hilarious.

File: src/backend/expungeservice/record_summarizer.py [hunk 2]
```diff
@@ -1,108 +1,38 @@
...
-        partially_eligible_cases = []
-        other_cases = []
-
         county_balances: Dict[str, float] = {}
         for case in record.cases:
             if not case.summary.location in county_balances.keys():
                 county_balances[case.summary.location] = case.summary.get_balance_due()
             else:
                 county_balances[case.summary.location] += case.summary.get_balance_due()
-
```
Comment (wittejm) [inline src/backend/expungeservice/record_summarizer.py:33]: Awesome awesome deleted section

**Comment by KentShikama** [comment]
@wittejm See https://github.com/codeforpdx/recordexpungPDX/pull/1204/commits/c293bbbddad72402a335bf1bf7c39e4ceef00517. I've gone ahead and merged this as I want to give @michaelzhang43 a chance to review in the morning.

**Comment by michaelzhang43** [comment]
I kind of like this 
Screen Shot 2020-06-10 at 11 26 08 PM

**Pull request merged**